java/client/src/main/java/glide/api/models/commands/FT/FTSearchOptions.java

```diff
@@ -0,0 +1,130 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.FT;
+
+import static glide.api.models.GlideString.gs;
+
+import glide.api.commands.servermodules.FT;
+import glide.api.models.GlideString;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Builder;
+import org.apache.commons.lang3.tuple.Pair;
+
+/** Mandatory parameters for {@link FT#search}. */
+@Builder
+public class FTSearchOptions {
+
+    @Builder.Default private final Map<GlideString, GlideString> identifiers = new HashMap<>();
```

Comment:
The value is alias in this and it is optional. Will the user provide null as input if they don't want to provide alias as input?

Reply:
I provide 2 options to add fields there - with and without alias:
```
addReturnField(String field)
addReturnField(String field, String alias)
```
see lines 86-104

Reply:
Got it. Thanks.